### PR TITLE
Controller menu

### PR DESCRIPTION
**Pull request by Sergeanur**

### CODE REVIEW

File: src/core/Frontend.cpp [hunk 1]
```diff
@@ -2280,6 +2300,9 @@ CMenuManager::DrawBackground(bool transitionCall)
 				break;
 			case MENUPAGE_SKIN_SELECT:
 			case MENUPAGE_KEYBOARD_CONTROLS:
+#ifdef GAMEPAD_MENU
+			case MENUPAGE_CONTROLLER_SETTINGS:
+#endif
```
Comment (erorcun) [inline src/core/Frontend.cpp:2305]: I think we can copy these values to cfo's noInvasiveBorders condition below, enable it in CCustomScreen and remove this #ifdef

File: src/core/MenuScreensCustom.cpp [hunk 2]
```diff
@@ -612,11 +626,13 @@ CMenuScreenCustom aScreens[] = {
 #else
 		MENUACTION_KEYBOARDCTRLS,"FEC_RED", {nil, SAVESLOT_NONE, MENUPAGE_KEYBOARD_CONTROLS}, 320, 150, MENUALIGN_CENTER,
 #endif
-#ifdef DETECT_JOYSTICK_MENU
+#ifdef GAMEPAD_MENU
+		MENUACTION_CHANGEMENU,	"FET_AGS", {nil, SAVESLOT_NONE, MENUPAGE_CONTROLLER_SETTINGS}, 0, 0, MENUALIGN_CENTER,
+#endif
+#ifdef DONT_TRUST_RECOGNIZED_JOYSTICKS
```
Comment (erorcun) [inline src/core/MenuScreensCustom.cpp:632]: This was changed to DETECT_JOYSTICK_MENU a few commits ago.

File: src/core/Frontend.cpp [hunk 3]
```diff
@@ -1074,7 +1089,7 @@ CMenuManager::DrawStandardMenus(bool activeScreen)
 				}
 
 				switch (aScreens[m_nCurrScreen].m_aEntries[i].m_Action) {
-#ifdef LEGACY_MENU_OPTIONS
+#if defined(LEGACY_MENU_OPTIONS) || defined(GAMEPAD_MENU)
```
Comment (erorcun) [inline src/core/Frontend.cpp:1092]: I think you can remove LEGACY_MENU_OPTIONS from all gamepad related things, they weren't visible anyway.

File: src/core/Frontend.cpp (hunk 1 above)
Comment (Sergeanur) [reply src/core/Frontend.cpp]: I've left it with the values that were there, it looks fine.

File: utils/gxt/spanish.txt [hunk 4]
```diff
@@ -13952,8 +13953,9 @@ Para ~h~recoger una bomba~w~, simplemente dirige el helicóptero RC hasta ella.
 [TEX3_37:SERG3]
 Si empujas ~h~el joystick analógico derecho hacia atrás~w~, aumentarás la velocidad del rotor y así ~h~ascenderá~w~.
 
-[TEX3_38:SERG3]
-Pulsa ~h~~k~~VEHICLE_ACCELERATE~~w~ para aumentar la velocidad del rotor y así ~h~ascender~w~.
+[TEX3_38:SERG3] { reVC update }
+{Pulsa ~h~~k~~VEHICLE_ACCELERATE~~w~ para aumentar la velocidad del rotor y así ~h~ascender~w~.}
+Si empujas ~h~el joystick analógico derecho hacia adelante~w~disminuye la velocidad del rotor aumenta haciendo que el helicóptero ~h~descienda.
```
Comment (IlDucci) [inline utils/gxt/spanish.txt:13958]: There's an extra unneeded "disminuye" in there and seems to miss a comma. It should read like this:
`Si empujas ~h~el joystick analógico derecho hacia adelante~w~ la velocidad del rotor aumenta, haciendo que el helicóptero ~h~descienda.`

Comment (Sergeanur) [reply utils/gxt/spanish.txt]: Fixed

**Pull request merged**